### ISSUE TITLE
ArgumentError when returning false from self.authorized?

### ISSUE THREAD

**Issue by gregawoods** [rmosolgo/graphql-ruby#1853]
I'm attempting to implement authorization according to the [guide here.](http://graphql-ruby.org/authorization/authorization.html)

> When you implement this method to return false, the query will be halted

However, when I return false from the `self.authorized` method what I get back is an `ArgumentError` complaining about a missing message or keyword.

Using graphql 1.8.9.

```ruby
class Types::BaseObject < GraphQL::Schema::Object
  def self.authorized?(object, context)
    false
  end
end
```

```
{
  "status": 500,
  "error": "Internal Server Error",
  "exception": "#<ArgumentError: GraphQL::UnauthorizedError requires either a message or keywords>",
  "traces": {
    "Application Trace": [
      {
        "id": 25,
        "trace": "app/controllers/graphql_controller.rb:12:in `execute'"
      }
    ],
    "Framework Trace": [
      {
        "id": 0,
        "trace": "graphql (1.8.9) lib/graphql/unauthorized_error.rb:15:in `initialize'"
      },
      {
        "id": 1,
        "trace": "graphql (1.8.9) lib/graphql/schema/object.rb:43:in `new'"
      },
      {
        "id": 2,
        "trace": "graphql (1.8.9) lib/graphql/schema/object.rb:43:in `block in authorized_new'"
      },
      {
        "id": 3,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:1013:in `after_lazy'"
      },
      {
        "id": 4,
        "trace": "graphql (1.8.9) lib/graphql/schema/object.rb:38:in `authorized_new'"
      },
      {
        "id": 5,
        "trace": "graphql (1.8.9) lib/graphql/schema/member/instrumentation.rb:30:in `before_query'"
      },
      {
        "id": 6,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:58:in `public_send'"
      },
      {
        "id": 7,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:58:in `block in call_hooks'"
      },
      {
        "id": 8,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:57:in `each'"
      },
      {
        "id": 9,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:57:in `call_hooks'"
      },
      {
        "id": 10,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:44:in `each_query_call_hooks'"
      },
      {
        "id": 11,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:27:in `block in apply_instrumenters'"
      },
      {
        "id": 12,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:70:in `call_hooks'"
      },
      {
        "id": 13,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:26:in `apply_instrumenters'"
      },
      {
        "id": 14,
        "trace": "graphql (1.8.9) lib/graphql/execution/multiplex.rb:166:in `instrument_and_analyze'"
      },
      {
        "id": 15,
        "trace": "graphql (1.8.9) lib/graphql/execution/multiplex.rb:68:in `block in run_queries'"
      },
      {
        "id": 16,
        "trace": "graphql (1.8.9) lib/graphql/tracing.rb:62:in `block in trace'"
      },
      {
        "id": 17,
        "trace": "graphql (1.8.9) lib/graphql/tracing.rb:76:in `call_tracers'"
      },
      {
        "id": 18,
        "trace": "graphql (1.8.9) lib/graphql/tracing.rb:62:in `trace'"
      },
      {
        "id": 19,
        "trace": "graphql (1.8.9) lib/graphql/execution/multiplex.rb:58:in `run_queries'"
      },
      {
        "id": 20,
        "trace": "graphql (1.8.9) lib/graphql/execution/multiplex.rb:48:in `run_all'"
      },
      {
        "id": 21,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:342:in `block in multiplex'"
      },
      {
        "id": 22,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:1079:in `with_definition_error_check'"
      },
      {
        "id": 23,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:341:in `multiplex'"
      },
      {
        "id": 24,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:318:in `execute'"
      },
      {
        "id": 26,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/basic_implicit_render.rb:4:in `send_action'"
      },
      {
        "id": 27,
        "trace": "actionpack (5.1.6) lib/abstract_controller/base.rb:186:in `process_action'"
      },
      {
        "id": 28,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/rendering.rb:30:in `process_action'"
      },
      {
        "id": 29,
        "trace": "actionpack (5.1.6) lib/abstract_controller/callbacks.rb:20:in `block in process_action'"
      },
      {
        "id": 30,
        "trace": "activesupport (5.1.6) lib/active_support/callbacks.rb:97:in `run_callbacks'"
      },
      {
        "id": 31,
        "trace": "actionpack (5.1.6) lib/abstract_controller/callbacks.rb:19:in `process_action'"
      },
      {
        "id": 32,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/rescue.rb:20:in `process_action'"
      },
      {
        "id": 33,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/instrumentation.rb:32:in `block in process_action'"
      },
      {
        "id": 34,
        "trace": "activesupport (5.1.6) lib/active_support/notifications.rb:166:in `block in instrument'"
      },
      {
        "id": 35,
        "trace": "activesupport (5.1.6) lib/active_support/notifications/instrumenter.rb:21:in `instrument'"
      },
      {
        "id": 36,
        "trace": "activesupport (5.1.6) lib/active_support/notifications.rb:166:in `instrument'"
      },
      {
        "id": 37,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/instrumentation.rb:30:in `process_action'"
      },
      {
        "id": 38,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/params_wrapper.rb:252:in `process_action'"
      },
      {
        "id": 39,
        "trace": "activerecord (5.1.6) lib/active_record/railties/controller_runtime.rb:22:in `process_action'"
      },
      {
        "id": 40,
        "trace": "actionpack (5.1.6) lib/abstract_controller/base.rb:124:in `process'"
      },
      {
        "id": 41,
        "trace": "actionview (5.1.6) lib/action_view/rendering.rb:30:in `process'"
      },
      {
        "id": 42,
        "trace": "actionpack (5.1.6) lib/action_controller/metal.rb:189:in `dispatch'"
      },
      {
        "id": 43,
        "trace": "actionpack (5.1.6) lib/action_controller/metal.rb:253:in `dispatch'"
      },
      {
        "id": 44,
        "trace": "actionpack (5.1.6) lib/action_dispatch/routing/route_set.rb:49:in `dispatch'"
      },
      {
        "id": 45,
        "trace": "actionpack (5.1.6) lib/action_dispatch/routing/route_set.rb:31:in `serve'"
      },
      {
        "id": 46,
        "trace": "actionpack (5.1.6) lib/action_dispatch/journey/router.rb:50:in `block in serve'"
      },
      {
        "id": 47,
        "trace": "actionpack (5.1.6) lib/action_dispatch/journey/router.rb:33:in `each'"
      },
      {
        "id": 48,
        "trace": "actionpack (5.1.6) lib/action_dispatch/journey/router.rb:33:in `serve'"
      },
      {
        "id": 49,
        "trace": "actionpack (5.1.6) lib/action_dispatch/routing/route_set.rb:844:in `call'"
      },
      {
        "id": 50,
        "trace": "rack (2.0.5) lib/rack/etag.rb:25:in `call'"
      },
      {
        "id": 51,
        "trace": "rack (2.0.5) lib/rack/conditional_get.rb:38:in `call'"
      },
      {
        "id": 52,
        "trace": "rack (2.0.5) lib/rack/head.rb:12:in `call'"
      },
      {
        "id": 53,
        "trace": "rack (2.0.5) lib/rack/session/abstract/id.rb:232:in `context'"
      },
      {
        "id": 54,
        "trace": "rack (2.0.5) lib/rack/session/abstract/id.rb:226:in `call'"
      },
      {
        "id": 55,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/cookies.rb:613:in `call'"
      },
      {
        "id": 56,
        "trace": "activerecord (5.1.6) lib/active_record/migration.rb:556:in `call'"
      },
      {
        "id": 57,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/callbacks.rb:26:in `block in call'"
      },
      {
        "id": 58,
        "trace": "activesupport (5.1.6) lib/active_support/callbacks.rb:97:in `run_callbacks'"
      },
      {
        "id": 59,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/callbacks.rb:24:in `call'"
      },
      {
        "id": 60,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/executor.rb:12:in `call'"
      },
      {
        "id": 61,
        "trace": "rollbar (2.17.0) lib/rollbar/middleware/rails/rollbar.rb:24:in `block in call'"
      },
      {
        "id": 62,
        "trace": "rollbar (2.17.0) lib/rollbar.rb:146:in `scoped'"
      },
      {
        "id": 63,
        "trace": "rollbar (2.17.0) lib/rollbar/middleware/rails/rollbar.rb:22:in `call'"
      },
      {
        "id": 64,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/debug_exceptions.rb:59:in `call'"
      },
      {
        "id": 65,
        "trace": "rollbar (2.17.0) lib/rollbar/middleware/rails/show_exceptions.rb:22:in `call_with_rollbar'"
      },
      {
        "id": 66,
        "trace": "web-console (3.6.2) lib/web_console/middleware.rb:135:in `call_app'"
      },
      {
        "id": 67,
        "trace": "web-console (3.6.2) lib/web_console/middleware.rb:22:in `block in call'"
      },
      {
        "id": 68,
        "trace": "web-console (3.6.2) lib/web_console/middleware.rb:20:in `catch'"
      },
      {
        "id": 69,
        "trace": "web-console (3.6.2) lib/web_console/middleware.rb:20:in `call'"
      },
      {
        "id": 70,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'"
      },
      {
        "id": 71,
        "trace": "railties (5.1.6) lib/rails/rack/logger.rb:36:in `call_app'"
      },
      {
        "id": 72,
        "trace": "railties (5.1.6) lib/rails/rack/logger.rb:24:in `block in call'"
      },
      {
        "id": 73,
        "trace": "activesupport (5.1.6) lib/active_support/tagged_logging.rb:69:in `block in tagged'"
      },
      {
        "id": 74,
        "trace": "activesupport (5.1.6) lib/active_support/tagged_logging.rb:26:in `tagged'"
      },
      {
        "id": 75,
        "trace": "activesupport (5.1.6) lib/active_support/tagged_logging.rb:69:in `tagged'"
      },
      {
        "id": 76,
        "trace": "railties (5.1.6) lib/rails/rack/logger.rb:24:in `call'"
      },
      {
        "id": 77,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'"
      },
      {
        "id": 78,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/request_id.rb:25:in `call'"
      },
      {
        "id": 79,
        "trace": "rack (2.0.5) lib/rack/method_override.rb:22:in `call'"
      },
      {
        "id": 80,
        "trace": "rack (2.0.5) lib/rack/runtime.rb:22:in `call'"
      },
      {
        "id": 81,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/executor.rb:12:in `call'"
      },
      {
        "id": 82,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/static.rb:125:in `call'"
      },
      {
        "id": 83,
        "trace": "rack (2.0.5) lib/rack/sendfile.rb:111:in `call'"
      },
      {
        "id": 84,
        "trace": "rack-cors (1.0.2) lib/rack/cors.rb:97:in `call'"
      },
      {
        "id": 85,
        "trace": "webpacker (3.5.3) lib/webpacker/dev_server_proxy.rb:22:in `perform_request'"
      },
      {
        "id": 86,
        "trace": "rack-proxy (0.6.4) lib/rack/proxy.rb:57:in `call'"
      },
      {
        "id": 87,
        "trace": "railties (5.1.6) lib/rails/engine.rb:522:in `call'"
      },
      {
        "id": 88,
        "trace": "puma (3.11.4) lib/puma/configuration.rb:225:in `call'"
      },
      {
        "id": 89,
        "trace": "puma (3.11.4) lib/puma/server.rb:632:in `handle_request'"
      },
      {
        "id": 90,
        "trace": "puma (3.11.4) lib/puma/server.rb:446:in `process_client'"
      },
      {
        "id": 91,
        "trace": "puma (3.11.4) lib/puma/server.rb:306:in `block in run'"
      },
      {
        "id": 92,
        "trace": "puma (3.11.4) lib/puma/thread_pool.rb:120:in `block in spawn_thread'"
      }
    ],
    "Full Trace": [
      {
        "id": 0,
        "trace": "graphql (1.8.9) lib/graphql/unauthorized_error.rb:15:in `initialize'"
      },
      {
        "id": 1,
        "trace": "graphql (1.8.9) lib/graphql/schema/object.rb:43:in `new'"
      },
      {
        "id": 2,
        "trace": "graphql (1.8.9) lib/graphql/schema/object.rb:43:in `block in authorized_new'"
      },
      {
        "id": 3,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:1013:in `after_lazy'"
      },
      {
        "id": 4,
        "trace": "graphql (1.8.9) lib/graphql/schema/object.rb:38:in `authorized_new'"
      },
      {
        "id": 5,
        "trace": "graphql (1.8.9) lib/graphql/schema/member/instrumentation.rb:30:in `before_query'"
      },
      {
        "id": 6,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:58:in `public_send'"
      },
      {
        "id": 7,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:58:in `block in call_hooks'"
      },
      {
        "id": 8,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:57:in `each'"
      },
      {
        "id": 9,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:57:in `call_hooks'"
      },
      {
        "id": 10,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:44:in `each_query_call_hooks'"
      },
      {
        "id": 11,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:27:in `block in apply_instrumenters'"
      },
      {
        "id": 12,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:70:in `call_hooks'"
      },
      {
        "id": 13,
        "trace": "graphql (1.8.9) lib/graphql/execution/instrumentation.rb:26:in `apply_instrumenters'"
      },
      {
        "id": 14,
        "trace": "graphql (1.8.9) lib/graphql/execution/multiplex.rb:166:in `instrument_and_analyze'"
      },
      {
        "id": 15,
        "trace": "graphql (1.8.9) lib/graphql/execution/multiplex.rb:68:in `block in run_queries'"
      },
      {
        "id": 16,
        "trace": "graphql (1.8.9) lib/graphql/tracing.rb:62:in `block in trace'"
      },
      {
        "id": 17,
        "trace": "graphql (1.8.9) lib/graphql/tracing.rb:76:in `call_tracers'"
      },
      {
        "id": 18,
        "trace": "graphql (1.8.9) lib/graphql/tracing.rb:62:in `trace'"
      },
      {
        "id": 19,
        "trace": "graphql (1.8.9) lib/graphql/execution/multiplex.rb:58:in `run_queries'"
      },
      {
        "id": 20,
        "trace": "graphql (1.8.9) lib/graphql/execution/multiplex.rb:48:in `run_all'"
      },
      {
        "id": 21,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:342:in `block in multiplex'"
      },
      {
        "id": 22,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:1079:in `with_definition_error_check'"
      },
      {
        "id": 23,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:341:in `multiplex'"
      },
      {
        "id": 24,
        "trace": "graphql (1.8.9) lib/graphql/schema.rb:318:in `execute'"
      },
      {
        "id": 25,
        "trace": "app/controllers/graphql_controller.rb:12:in `execute'"
      },
      {
        "id": 26,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/basic_implicit_render.rb:4:in `send_action'"
      },
      {
        "id": 27,
        "trace": "actionpack (5.1.6) lib/abstract_controller/base.rb:186:in `process_action'"
      },
      {
        "id": 28,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/rendering.rb:30:in `process_action'"
      },
      {
        "id": 29,
        "trace": "actionpack (5.1.6) lib/abstract_controller/callbacks.rb:20:in `block in process_action'"
      },
      {
        "id": 30,
        "trace": "activesupport (5.1.6) lib/active_support/callbacks.rb:97:in `run_callbacks'"
      },
      {
        "id": 31,
        "trace": "actionpack (5.1.6) lib/abstract_controller/callbacks.rb:19:in `process_action'"
      },
      {
        "id": 32,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/rescue.rb:20:in `process_action'"
      },
      {
        "id": 33,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/instrumentation.rb:32:in `block in process_action'"
      },
      {
        "id": 34,
        "trace": "activesupport (5.1.6) lib/active_support/notifications.rb:166:in `block in instrument'"
      },
      {
        "id": 35,
        "trace": "activesupport (5.1.6) lib/active_support/notifications/instrumenter.rb:21:in `instrument'"
      },
      {
        "id": 36,
        "trace": "activesupport (5.1.6) lib/active_support/notifications.rb:166:in `instrument'"
      },
      {
        "id": 37,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/instrumentation.rb:30:in `process_action'"
      },
      {
        "id": 38,
        "trace": "actionpack (5.1.6) lib/action_controller/metal/params_wrapper.rb:252:in `process_action'"
      },
      {
        "id": 39,
        "trace": "activerecord (5.1.6) lib/active_record/railties/controller_runtime.rb:22:in `process_action'"
      },
      {
        "id": 40,
        "trace": "actionpack (5.1.6) lib/abstract_controller/base.rb:124:in `process'"
      },
      {
        "id": 41,
        "trace": "actionview (5.1.6) lib/action_view/rendering.rb:30:in `process'"
      },
      {
        "id": 42,
        "trace": "actionpack (5.1.6) lib/action_controller/metal.rb:189:in `dispatch'"
      },
      {
        "id": 43,
        "trace": "actionpack (5.1.6) lib/action_controller/metal.rb:253:in `dispatch'"
      },
      {
        "id": 44,
        "trace": "actionpack (5.1.6) lib/action_dispatch/routing/route_set.rb:49:in `dispatch'"
      },
      {
        "id": 45,
        "trace": "actionpack (5.1.6) lib/action_dispatch/routing/route_set.rb:31:in `serve'"
      },
      {
        "id": 46,
        "trace": "actionpack (5.1.6) lib/action_dispatch/journey/router.rb:50:in `block in serve'"
      },
      {
        "id": 47,
        "trace": "actionpack (5.1.6) lib/action_dispatch/journey/router.rb:33:in `each'"
      },
      {
        "id": 48,
        "trace": "actionpack (5.1.6) lib/action_dispatch/journey/router.rb:33:in `serve'"
      },
      {
        "id": 49,
        "trace": "actionpack (5.1.6) lib/action_dispatch/routing/route_set.rb:844:in `call'"
      },
      {
        "id": 50,
        "trace": "rack (2.0.5) lib/rack/etag.rb:25:in `call'"
      },
      {
        "id": 51,
        "trace": "rack (2.0.5) lib/rack/conditional_get.rb:38:in `call'"
      },
      {
        "id": 52,
        "trace": "rack (2.0.5) lib/rack/head.rb:12:in `call'"
      },
      {
        "id": 53,
        "trace": "rack (2.0.5) lib/rack/session/abstract/id.rb:232:in `context'"
      },
      {
        "id": 54,
        "trace": "rack (2.0.5) lib/rack/session/abstract/id.rb:226:in `call'"
      },
      {
        "id": 55,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/cookies.rb:613:in `call'"
      },
      {
        "id": 56,
        "trace": "activerecord (5.1.6) lib/active_record/migration.rb:556:in `call'"
      },
      {
        "id": 57,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/callbacks.rb:26:in `block in call'"
      },
      {
        "id": 58,
        "trace": "activesupport (5.1.6) lib/active_support/callbacks.rb:97:in `run_callbacks'"
      },
      {
        "id": 59,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/callbacks.rb:24:in `call'"
      },
      {
        "id": 60,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/executor.rb:12:in `call'"
      },
      {
        "id": 61,
        "trace": "rollbar (2.17.0) lib/rollbar/middleware/rails/rollbar.rb:24:in `block in call'"
      },
      {
        "id": 62,
        "trace": "rollbar (2.17.0) lib/rollbar.rb:146:in `scoped'"
      },
      {
        "id": 63,
        "trace": "rollbar (2.17.0) lib/rollbar/middleware/rails/rollbar.rb:22:in `call'"
      },
      {
        "id": 64,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/debug_exceptions.rb:59:in `call'"
      },
      {
        "id": 65,
        "trace": "rollbar (2.17.0) lib/rollbar/middleware/rails/show_exceptions.rb:22:in `call_with_rollbar'"
      },
      {
        "id": 66,
        "trace": "web-console (3.6.2) lib/web_console/middleware.rb:135:in `call_app'"
      },
      {
        "id": 67,
        "trace": "web-console (3.6.2) lib/web_console/middleware.rb:22:in `block in call'"
      },
      {
        "id": 68,
        "trace": "web-console (3.6.2) lib/web_console/middleware.rb:20:in `catch'"
      },
      {
        "id": 69,
        "trace": "web-console (3.6.2) lib/web_console/middleware.rb:20:in `call'"
      },
      {
        "id": 70,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'"
      },
      {
        "id": 71,
        "trace": "railties (5.1.6) lib/rails/rack/logger.rb:36:in `call_app'"
      },
      {
        "id": 72,
        "trace": "railties (5.1.6) lib/rails/rack/logger.rb:24:in `block in call'"
      },
      {
        "id": 73,
        "trace": "activesupport (5.1.6) lib/active_support/tagged_logging.rb:69:in `block in tagged'"
      },
      {
        "id": 74,
        "trace": "activesupport (5.1.6) lib/active_support/tagged_logging.rb:26:in `tagged'"
      },
      {
        "id": 75,
        "trace": "activesupport (5.1.6) lib/active_support/tagged_logging.rb:69:in `tagged'"
      },
      {
        "id": 76,
        "trace": "railties (5.1.6) lib/rails/rack/logger.rb:24:in `call'"
      },
      {
        "id": 77,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'"
      },
      {
        "id": 78,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/request_id.rb:25:in `call'"
      },
      {
        "id": 79,
        "trace": "rack (2.0.5) lib/rack/method_override.rb:22:in `call'"
      },
      {
        "id": 80,
        "trace": "rack (2.0.5) lib/rack/runtime.rb:22:in `call'"
      },
      {
        "id": 81,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/executor.rb:12:in `call'"
      },
      {
        "id": 82,
        "trace": "actionpack (5.1.6) lib/action_dispatch/middleware/static.rb:125:in `call'"
      },
      {
        "id": 83,
        "trace": "rack (2.0.5) lib/rack/sendfile.rb:111:in `call'"
      },
      {
        "id": 84,
        "trace": "rack-cors (1.0.2) lib/rack/cors.rb:97:in `call'"
      },
      {
        "id": 85,
        "trace": "webpacker (3.5.3) lib/webpacker/dev_server_proxy.rb:22:in `perform_request'"
      },
      {
        "id": 86,
        "trace": "rack-proxy (0.6.4) lib/rack/proxy.rb:57:in `call'"
      },
      {
        "id": 87,
        "trace": "railties (5.1.6) lib/rails/engine.rb:522:in `call'"
      },
      {
        "id": 88,
        "trace": "puma (3.11.4) lib/puma/configuration.rb:225:in `call'"
      },
      {
        "id": 89,
        "trace": "puma (3.11.4) lib/puma/server.rb:632:in `handle_request'"
      },
      {
        "id": 90,
        "trace": "puma (3.11.4) lib/puma/server.rb:446:in `process_client'"
      },
      {
        "id": 91,
        "trace": "puma (3.11.4) lib/puma/server.rb:306:in `block in run'"
      },
      {
        "id": 92,
        "trace": "puma (3.11.4) lib/puma/thread_pool.rb:120:in `block in spawn_thread'"
      }
    ]
  }
}```

**Comment by rmosolgo** [comment]
Weird bug! Thanks for the detailed report. It looks like some code that handles the returned `false` needs to be updated.

**Comment by eapache** [comment]
I think this is what happens if: you `authorized?` as `false` on your query root, and you don't provide a root value for your query, since then `object` will be null.

**Comment by rmosolgo** [comment]
😯

**Comment by jurre** [comment]
So, is there another recommended way to handle this?

**Comment by jurre** [comment]
So, if I understand correctly we should include a message here:

https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/schema/object.rb#L49

Happy to open a PR if that helps

**Comment by rmosolgo** [comment]
Thanks for triaging this @eapache, you were spot on! Should be fixed in 1.9.3, please let me know if you have any more trouble.
